app/sensors/DropdownList.js

```diff
@@ -138,9 +138,27 @@ export default class DropdownList extends Component {
 				}
 			};
 		} else if (value) {
+			// queryFormat should not affect SingleDropdownList
+			if (this.props.queryFormat === "or" || !this.props.multipleSelect) {
+				return {
+					[this.type]: {
+						[this.props.appbaseField]: value
+					}
+				};
+			}
+
+			// for queryFormat = "and"
+			const queryArray = value.map(item => (
+				{
+					[this.type]: {
+						[this.props.appbaseField]: [item]
+					}
+				}
+			));
```

Comment:
I think it would be much simpler and easier to read, if we keep:

`if (this.props.queryFormat === "and" && this.props.multipleSelect)` for the multi-terms-query array and keep the `or` query as a default here.